### PR TITLE
New blocks

### PR DESCRIPTION
**Pull request by truckutter**

### CODE REVIEW

File: src/pocketmine/nbt/tag/IntArray.php [hunk 1]
```diff
@@ -32,12 +32,9 @@ public function getType(){
 	}
 
 	public function read(NBT $nbt){
-		$this->value = [];
+		 [];
```
Comment (PEMapModder) [inline src/pocketmine/nbt/tag/IntArray.php]: What?

**Comment by matcracker** [comment]
Cool work................................. ._.

File: src/pocketmine/nbt/tag/IntArray.php (hunk 1 above)
Comment (iJoshuaHD) [inline src/pocketmine/nbt/tag/IntArray.php]: lol wtf xD

**Comment by PEMapModder** [comment]
The problem is, this isn't his pull request at all. He only tries to merge things from a branch to another branch while he didn't make the commits. **This should be closed.**

**Comment by sekjun9878** [comment]
Merging between branches of the PocketMine-MP Repository should only be handled by PocketMine Git Team Members. Closing Pull Request.

**Pull request closed without merging**